data/talkactions/scripts/create_item.lua

```diff
@@ -28,11 +35,7 @@ function onSay(player, words, param)
 			count = math.max(0, count)
 		end
 	else
-		if not itemType:isFluidContainer() then
-			count = 1
-		else
-			count = 0
-		end
+		count = itemType:isFluidContainer() and 0 or 1
```

Comment:
A suggestion: create a local function `getDefaultCount` and return the count for each situation. Ternary operations are ugly in Lua.